src/vm/arm/asmhelpers.S

```diff
@@ -1354,7 +1354,42 @@ DelayLoad_Helper\suffix:
 
 #endif // FEATURE_READYTORUN
 
+#define CONTEXT_OF_R4	(20)
```

Comment:
It might be good to use the same technique as src/vm/amd64/asmconstants.h to define them properly and have assert to ensure they are correct.